gui/wxpython/gui_core/gselect.py

```diff
@@ -2652,6 +2663,27 @@ def _getPGDBRasters(self, dsn):
         Debug.msg(3, f"GdalSelect._getPGDBRasters(): return {rasters}")
         return rasters
 
+    def _getRasterliteDBRasters(self, dsn):
+        """Get Rasterlite DB rasters
+
+        :param str dsn: Rasterlite DB data source name
+
+        :return list: list of Rasterlite DB rasters
+        """
+        try:
+            from osgeo import gdal
+        except ImportError:
+            GError(
+                parent=self,
+                message=_("The GDAL library is missing. Please install it."),
```

Comment:
```suggestion
                message=_("The Python osgeo package is missing. Please install it."),
```

Reply:
Fixed with https://github.com/OSGeo/grass/pull/2513/commits/0f2fccaa1efe22293793f4d22bfd787757fd980f.

Reply:
Confusingly, the package is called GDAL as afar as I can tell: https://pypi.org/project/GDAL/

Reply:
Fixed with https://github.com/OSGeo/grass/pull/2513/commits/685cb7af6288604fd4acd84549f5befc6d9402a6.